src/components/dashboard/FeedList.js

```diff
@@ -54,6 +45,10 @@ const getItemLayout = (_: any, index: number) => {
   }
 }
 
+const Item = React.memo(({ item, handleFeedSelection }) => {
```

Comment:
1. key doesn't have effect here, it should me applied in the place where we rendering components array using `.map()`
2. why do we need this extra component ? FeedListItem is already pure component

Reply:
1. key is required for the flatlast items
2. it could be that we dont need it, i'll test

Reply:
2. tested an no. i dont know why, but this way it doesnt re-render the whole feed list items when we add items to the feed on scroll. using FeedLiistItem directly, causes re-render of existting items. with Item it only renders new feed items